modules/learn/pages/security/authentication-overview.adoc

```diff
@@ -77,6 +77,34 @@ For more information on the using the REST API to configure security settings, s
 
 For cases that do not use SCRAM-SHA, such as those that implement TLS network encryption, it is recommended to _disable_ all variations of SCRAM-SHA; as this will prevent the SCRAM-SHA PBKDF2 hashes from being generated, when Argon2ID provides stronger security for those credentials.
 
+
+[#password-hash-migration]
+=== Automatic Password Hash Migration
```

Comment:
Probably it makes sense to add a warning in this paragraph. When migrating to argon, please keep in mind that by design Argon2id consumes significantly more system resources such as CPU and memory than sha1. If your server handles high rate of authentications, migration to argon can lead to a significant spike in cpu and memory consumption.

Reply:
Added a note after first mention of Argon2id. Also added it to the reference page for the passwordHashAlg setting.